src/transactionsFactories/smartContractTransactionsFactory.ts

```diff
@@ -146,7 +166,11 @@ export class SmartContractTransactionsFactory {
         const args = options.args || [];
         const metadata = new CodeMetadata(isUpgradeable, isReadable, isPayable, isPayableBySmartContract);
 
-        let parts = ["upgradeContract", byteArrayToHex(options.bytecode), metadata.toString()];
+        let parts = [
```

Comment:
Maybe here, keep `byteArrayToHex`, as before (simpler)?

Reply:
Reverted the changes.